sections/shared/modals/UniswapModal/UniswapModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { Synths } from '@synthetixio/contracts-interface';
+import { SwapWidget, Theme, TokenInfo } from '@uniswap/widgets';
+import '@uniswap/widgets/dist/fonts.css';
+
+import BaseModal from 'components/BaseModal';
+import Connector from 'containers/Connector';
+import { FC } from 'react';
+import styled from 'styled-components';
+import { getInfuraRpcURL } from 'utils/infura';
+
+import DEFAULT_TOKEN_LIST from './defaultTokenList.json';
+
+// special referece on the uniswap widget
+// represents the Native Token of the current chain (typically ETH)
+const NATIVE = 'NATIVE';
+
+type UniswapModalProps = {
+	onDismiss: () => void;
+	isOpen?: boolean;
+	tokenList?: TokenInfo[];
+	inputTokenAddress?: string;
+	outputTokenAddress?: string;
+};
+
+const theme: Theme = {
+	primary: '#ECE8E3', // primary white
+	secondary: '#C3C0BA', // tertiary grey
+	interactive: '#C3C0BA', // tertiary grey
+	container: '#1E1D1D', // cell gradient
+	module: '#474747', // custom
+	accent: '#C9975B',
+	dialog: '#1E1D1D', // cell gradient
```

Comment:
Can't we import these from the selected theme file? When we implement themes this will always show using our dark theme, rather than inheriting the selected style.